### PR TITLE
chore: Remove any usage from storefront and make lintable

### LINKED ISSUE #225: Remove use of `any` from storefront

### PR DESCRIPTION
**Pull request by Thuneer**
Resolves #225

Removed all `any` uses from Storefront. Will come back at a later time and make types better and more reusable.

### CODE REVIEW

File: .hintrc
Comment (mimarz) [inline .hintrc:1]: This looks to be a configuration file for linter and hiner [Webhint](https://webhint.io/)? Was there a specific reason for using this? 

We are using [ESLint](https://eslint.org/) for this purpose and we should not need to use two different tools for solving the same problem.

**Review by mimarz — CHANGES_REQUESTED** [review]
Excellent work on removing the anys and getting some proper type safety in there! <3

File: .hintrc
Comment (Thuneer) [reply .hintrc:1]: I think I misclicked when I was using the "Quick fix" feature in VSCode. I removed the file in the next commit.

Comment (mimarz) [reply .hintrc:1]: huh, weird. Never heard of it before. Looks like its maybe something Microsoft is trying to push now 🙈

**Review by mimarz — APPROVED** [review]
How does it feel? Good amount of cleaning up done now! 🧹 👍

**Pull request merged**